google/cloud/pubsub/internal/message_batch.h

```diff
@@ -39,7 +39,7 @@ class MessageBatch {
 
   // Saves the message from the Publish call. Invoked in
   // `BatchingPublisherConnection::Publish(...)`.
-  virtual void SaveMessage(pubsub::Message m) = 0;
+  virtual void AddMessage(pubsub::Message m) = 0;
```

Comment:
While you are changing the signature, this should also be a `const&`.

Reply:
Already was working on it when I saw this comment. Nice.